sdks/go/pkg/beam/core/graph/edge.go

```diff
@@ -342,19 +342,18 @@ func NewCombine(g *Graph, s *Scope, u *CombineFn, in *Node) (*MultiEdge, error)
 		// However, the outbound type will be W<KV<A,O>> (where O is the output
```

Comment:
<!--new_thread; commit:0a7f1ff616d0ac0e00216a8ec85c983732017718; resolved:0-->
Are the W<>s in these paragraphs left intentionally?

Reply:
Nope. Good catch!